test/e2e_flags.go

```diff
@@ -99,6 +100,8 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 		"Set this flag to add extra HTTP request headers sent to the testing deployed KServices. "+
 			"Format: -request-headers=key1,value1,key2,value2")
 
+	flag.StringVar(&f.IngressClass, "ingress-class", "",
+		"Set this flag to indicate which ingress used for serving.")
```

Comment:
"Set this flag to indicate which ingress used for serving." is not correct, is it?
The ingress used for serving is still determined by the configuration in `config-network` and this flag does not control it.

Reply:
Agree with you, I'll try to make it clear and not ambiguous.